src/phoenix/server/api/helpers/jsonschema.py

```diff
@@ -114,7 +115,18 @@ def validate_json_schema_object_definition(schema: dict[str, Any]) -> dict[str,
     return schema
 
 
-# Pydantic type with built-in validation for JSON schemas
-JSONSchemaObjectDefinition: TypeAlias = Annotated[
-    dict[str, Any], AfterValidator(validate_json_schema_object_definition)
+JSONSchemaDraft7ObjectSchemaContent: TypeAlias = Annotated[
+    dict[str, Any],
+    AfterValidator(validate_json_schema_draft_7_object),
+]
+
+
+class JSONSchemaDraft7ObjectSchema(BaseModel):
+    type: Literal["json-schema-draft-7-object-schema"]
+    content: JSONSchemaDraft7ObjectSchemaContent
```

Comment:
nit. `json` is probably more descriptive in this setting
```suggestion
    json_: JSONSchemaDraft7ObjectSchemaContent
```